File: app/src/main/java/com/casebeaumonde/fragments/HireExpert/response/SendInvitationResponse.java
package com.casebeaumonde.fragments.HireExpert.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class SendInvitationResponse {

@SerializedName("code")
@Expose
private String code;

public String getCode() {
return code;
}

public void setCode(String code) {
this.code = code;
}

}

File: app/src/main/java/com/casebeaumonde/fragments/HireExpert/response/SendInvitationResponse.kt
package com.casebeaumonde.fragments.HireExpert.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class SendInvitationResponse {

    @SerializedName("code")
    @Expose
    private var code: String? = null

    fun getCode(): String? {
        return code
    }

    fun setCode(code: String?) {
        this.code = code
    }

}